CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ export(TARGETS uvc
   FILE "${PROJECT_BINARY_DIR}/libuvcTargets.cmake")
 export(PACKAGE libuvc)
 
-set(CONF_INCLUDE_DIR "${CMAKE_INSTALL_PREFIX}/include")
+set(CONF_INCLUDE_DIR "${CMAKE_INSTALL_PREFIX}/include;${LIBUSB_INCLUDE_DIRS}")
```

Comment:
Makes sense.

However I can't test it now. Feel free to close the PR for now, I can re-open it in the future if still needed.